BlazorServer/Startup.cs

```diff
@@ -120,6 +124,7 @@ public static void LogWrite(string message)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+            Syncfusion.Licensing.SyncfusionLicenseProvider.RegisterLicense("NDQ0NzA0QDMxMzkyZTMxMmUzMGdaWENEdFBGbFZWYy9KWGF3aFZ6NmhFcnVZNkFvbHZnQ1pBNGVkMjAxNTg9");
```

Comment:
This looking really sketchy.

In general its recommended to include secret keys in the source code :disappointed: